ArticleTemplates/assets/js/bootstraps/liveblog.js

```diff
@@ -131,18 +131,32 @@ function onLiveMoreClick(liveMoreElem) {
 }
 
 function setupTryLive() {
-    const elems = document.getElementsByClassName('live-promo__button');
+    const elems = document.getElementsByClassName('block--live-promo');
```

Comment:
Is this remote configurable on how many "try live" blocks we show?

Reply:
There should only ever be one on any liveblog, really. I'm happy to change this line to just pick the item zero.

Reply:
could do `const elem = document.querySelector('.block--live-promo');` ?